network-api/networkapi/buyersguide/pagemodels/product_update.py

```diff
@@ -2,13 +2,12 @@
 
 from wagtail.admin.edit_handlers import FieldPanel
 
-from wagtail.snippets.models import register_snippet
 from wagtail.search import index
 
 
-@register_snippet
 class Update(index.Indexed, models.Model):
-    # TODO: Delete this model.
+    # TODO: Delete this model. We'll need to do some migration surgery for this because `buyersguide.Update`
+    # is being used in the 0001_initial.py migration.
```

Comment:
I opened a ticket to do this properly so we can land this PR early. Follow up ticket here: #6547